astropy/io/fits/tests/test_fitsheader.py

```diff
@@ -0,0 +1,122 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import sys
+
+import numpy as np
+import re
+
+from . import FitsTestCase
+from ..hdu import PrimaryHDU
+from ..scripts import fitsheader
+from ....tests.helper import catch_warnings
+from ....tests.helper import pytest
```

Comment:
Could you rebase on master, and import pytest directly (the bundle version was removed recently).